bin/load-module-esm-support.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var dirnamePath = path.dirname;
+var extnamePath = path.extname;
+var resolvePath = path.resolve;
+
+var cwd = process.cwd();
+var isModulePackage = getPackageType() === 'module';
+
+function getPackageType() {
+    var dir = __dirname;
```

Comment:
It looks like you are determining the `type` which applies to `node_modules/tape/bin/load-module-esm-support.js`.  The reason `get-package-type` takes a filename argument is because each directory can be different even within a package.

Reply:
this was meant to be cwd, not __dirname... checking for every file felt like unneeded extra work but it probably needs to be done. and i see your code caches the results so shouldnt be too much overhead.